kubernetes-aws-yaml/Pulumi.yaml

```diff
@@ -0,0 +1,57 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: yaml
+template:
+  description: A YAML document to create a Kubernetes cluster on AWS
+  important: true
+  config:
+    aws:region:
+      description: The AWS region to deploy into
+      default: us-west-2
+configuration:
+  minClusterSize:
+    type: Number
+    default: 3
+  maxClusterSize:
+    type: Number
+    default: 6
+  desiredClusterSize:
+    type: Number
+    default: 3
+  eksNodeInstanceType:
+    type: String
+    default: t2.medium
+  vpcNetworkCidr:
+    type: String
+    default: 10.0.0.0/16
```

Comment:
Would it make sense to prompt for any of these (or some subset of them) during `pulumi new`? Seems like that might be nice, but I defer to you on it. If you think so too, we should duplicate them in under `template.config` (and add some descriptions).

Reply:
Fixed in 8f2808f. All languages (YAML, Go, Python, TypeScript) now prompt the user for these values when using `pulumi new`.